Library/Formula/h2.rb

```diff
@@ -1,10 +1,8 @@
-require 'formula'
```

Comment:
If removing this can you fix up everything else flagged by `brew audit --strict`? Thanks